pennylane/qchem/openfermion_obs.py

```diff
@@ -837,6 +839,13 @@ def molecular_hamiltonian(
 
     |
 
+    .. warning::
+
+        If this function is called with a :code:`grouping_type` while :code:`enable_new_opmath()` is active, then
+        an arithmetic operator will be returned and the grouping will be ignored.
+
+    |
+
```

Comment:
`enable_new_opmath()` is a new argument for `molecular_hamiltonian`?

Reply:
Nope, it is a function which sets the global variable to determine if dunder methods return arithmetic operators or hamiltonians.